.gitignore

```
@@ -33,3 +33,9 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+
+#ignoring node modules
+/node_modules/
+
+#ignore .idea
+/.idea/
```

Also add package-log.json
```suggestion
.idea/
/package-log.json
```